### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by JhanaviRathod**
All Spring security and jwt token with login and logout feature api's

### CODE REVIEW

File: persistence/src/main/java/com/persistence/model/auth/RefreshToken.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
+@Table(name="refresh_token")
+public class RefreshToken {
```
Comment (piyushsoni21) [inline persistence/src/main/java/com/persistence/model/auth/RefreshToken.java:23]: table name needs to be changed

File: persistence/src/main/java/com/persistence/model/auth/User.java [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+import java.util.Collection;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
+@Table(name="base_user")
+public class User implements UserDetails {
```
Comment (piyushsoni21) [inline persistence/src/main/java/com/persistence/model/auth/User.java:25]: table name is not correct it should be user_details or user_info

File: common/src/main/java/com/jobexecution/service/exception/BadRequestException.java [hunk 3]
```diff
@@ -0,0 +1,24 @@
+package com.jobexecution.service.exception;
+
+import lombok.Getter;
+import org.springframework.http.HttpStatus;
+
+@Getter
```
Comment (piyushsoni21) [inline common/src/main/java/com/jobexecution/service/exception/BadRequestException.java:6]: Remove this

File: persistence/src/main/java/com/persistence/model/auth/RefreshToken.java (hunk 1 above)
Comment (JhanaviRathod) [reply persistence/src/main/java/com/persistence/model/auth/RefreshToken.java:23]: @piyushsoni21 Please suggest the table name

File: common/src/main/java/com/jobexecution/service/exception/BadRequestException.java (hunk 3 above)
Comment (JhanaviRathod) [reply common/src/main/java/com/jobexecution/service/exception/BadRequestException.java:6]: @piyushsoni21 What needs to be removed?
`@Getter` annotation?

**Comment by piyushsoni21** [comment]
Resolved..Let's merge  this PR

**Pull request closed without merging**